rfcs/2021-09-08-motion-design-easing.md

```diff
@@ -0,0 +1,151 @@
+---
+Start Date: 09-08-2021
+RFC PR: NA
+Blade Issue: NA
+---
+# Easings - Motion Design <!-- omit in toc -->
+
+### Table Of Contents <!-- omit in toc -->
+- [Summary](#summary)
+- [Basic Example](#basic-example)
+- [Motivation](#motivation)
+- [Detailed Design](#detailed-design)
+- [Drawbacks/Constraints](#drawbacksconstraints)
+- [Alternatives](#alternatives)
+- [Adoption Strategy](#adoption-strategy)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+# Summary
+Easing refers to the way in which a motion proceeds between two states. You can think of easing as acceleration or deceleration. An object that moves from one side of the stage to the other side can start off slowly, then build up speed, and then stop suddenly.
+
+
+
+# Basic Example
+
+
+<img src="images/motion/easing/image14.gif" alt="bezier curves" > <br><br>
+
+
+### Basic Easing Style types: 
+Linear, Ease-in, Ease-out, Ease-in-out
+it can be infinite. Ref: https://easings.net/
+<br><br>
+
+### Easing value can be represented in multiple forms depending on the platform it is used:
+
+| Platform | Protocol |
+|---|---|
+| Android | FastOutSlowInInterpolator | 
+| CSS | cubic-bezier(0.4, 0.0, 0.2, 1) | 
+| Flutter| standardEasing| 
+| iOS| [[CAMediaTimingFunction alloc] <br> initWithControlPoints:0.4f:0.0f:0.2f:1.0f] | 
+| After effects | outgoing 20%, incoming 62% | 
+
+
+# Motivation
+### Why are we doing this?
+Easing is one of the elements of motion (out of 3 core actions, duration).
+ 
+* Strictly linear movement appears unnatural to the human eye and we are trying to make it pleasing to users eyes.
+* To maintain the consistency & branding throughout our motion styles so that it looks familiar to users and doesn't look out of the world. Defining these properties will help us in guiding the designers and communicating with the developers. 
+* Motion helps users to reach their goals and makes it self-explanatory, and these easing curves will help us in achieving the motion stylesand self-explanatory motion hints.
+* Defining the easing styles will help in guiding designers and communicating with developers.
+
+
+### What is the expected outcome?
+We shall be able to answer following questions as a conclusion to this RFC
+1. What easings we'll be using for any motion?
+2. How will we ensure accessibility in terms of motion?
+3. How all easings will be used to elements or components.
+4. How/where we will store the different easings as design tokens?
+
+# Detailed Design
+## Easing Types
+Based on motion purposes we need 4 types of easings-
```

Comment:
can we also add examples with each of the 4 easings for visual relation? Right now it's difficult to visualise from just text

Reply:
Examples are there, just scroll lil down (Final easings section). Should I add here as well for easy understanding?